packages/atlas/lib/src/circle.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/foundation.dart';
+import 'package:atlas/atlas.dart';
+import 'package:flutter/material.dart';
+
+/// `Circle` with specified center/radius on the map
+class Circle {
+  /// Uniquely identifies a `Circle`.
+  final String id;
+
+  /// The location where the `Circle` is centered. Is represented as `LatLng`.
+  final LatLng center;
+
+  /// The radius of the `Circle`.
+  final double radiusInMeters;
+
+  /// Optional FillColor used to color the area inside the `Circle`.
+  final Color fillColor;
+
+  /// Optional StrokeColor used to color the boundary of the `Circle`.
+  final Color strokeColor;
+
+  /// The z-index of the `Circle`, used to determine relative drawing order of
+  /// map overlays.
+  ///
+  /// Lower values means drawn earlier, and thus appearing to be closer to the surface of the Earth.
+  final double zIndex;
+
+  const Circle({
+    @required this.id,
+    @required this.center,
+    @required this.radiusInMeters,
+    this.fillColor,
```

Comment:
Shouldn't be `fillColor` and `strokeColor` also required, or assigned to a default color? Otherwise, we would be doing the computation of painting without painting anything, right?

Reply:
That makes sense for sure. **Black** for the `strokeColor` and **Transparent** for `fillColor`?

Reply:
Yep, that works ;)